home/tests/test_models.py

```diff
@@ -358,6 +368,25 @@ def test_for_missing_migrations(self):
             "There are missing migrations:\n %s" % output.getvalue(),
         )
 
+    @override_settings(WHATSAPP_CREATE_TEMPLATES=True)
+    @mock.patch("home.models.create_whatsapp_template")
+    @pytest.mark.xfail(
+        reason="This fails because we can't get locale to create the page, "
+        "these tests will be changed once whatsapp templates are separated."
+    )
```

Comment:
Note for other reviewers: We're replacing these tests with better ones (that don't use mocks, to start with) in #277. It's not worth duplicating that work here.